hpcflow/api.py

```diff
@@ -473,7 +474,7 @@ def update_config(name, value, config_dir=None):
 def cloud_connect(provider, config_dir=None):
```

Comment:
We will need to re-think this function. (I thought I fixed it with the change I made - but obviously that doesn't actually help!)

Reply:
I cant remember exactly what was wrong with this but I notice that in that function
`provider.check_access()`
the provider object is not instantiated. I have changed this to:
`provider().check_access()`

In testing now it seems to work as expected.